tests/integration_tests/functional/test_cpu_features_aarch64.py

```diff
@@ -53,7 +72,25 @@ def _check_cpu_features_arm(test_microvm, template_name=None):
         case "c7g.metal":
             match template_name:
                 case "aarch64_remove_ssbs":
-                    expected_cpu_features["Flags"] = DEFAULT_G3_FEATURES_NO_SSBS
+                    match kernel:
```

Comment:
nit: that is getting a bit hard to read. Would it be possible to replace with a LUT?

Reply:
Merged all `switch`es into one. Should be better now